[Bug Report]: Fail to build Integration Tests at macOS

### Prerequisites

- [X] I have searched [issues](https://github.com/Farfetch/kafkaflow/issues) to ensure it has not already been reported

### Description

Integration Tests fail to build at macOS.

### Steps to reproduce

Run the following command: 
`dotnet pack src/KafkaFlow.sln` 

### Expected behavior

Build with success. 
Integration Tests pass. 

### Actual behavior

Integration Tests fail to build at macOS.

```
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605: Warning As Error: Detected package downgrade: librdkafka.redist from 2.1.1 to 1.9.2. Reference the package directly from the project to select a different version.  [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605:  KafkaFlow.IntegrationTests -> KafkaFlow.Serializer.SchemaRegistry.ConfluentAvro -> Confluent.Kafka 2.1.1 -> librdkafka.redist (>= 2.1.1)  [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605:  KafkaFlow.IntegrationTests -> librdkafka.redist (>= 1.9.2) [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605: Warning As Error: Detected package downgrade: librdkafka.redist from 2.1.1 to 1.9.2. Reference the package directly from the project to select a different version.  [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605:  KafkaFlow.IntegrationTests -> KafkaFlow.Serializer.SchemaRegistry.ConfluentProtobuf -> Confluent.Kafka 2.1.1 -> librdkafka.redist (>= 2.1.1)  [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605:  KafkaFlow.IntegrationTests -> librdkafka.redist (>= 1.9.2) [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605: Warning As Error: Detected package downgrade: librdkafka.redist from 2.1.1 to 1.9.2. Reference the package directly from the project to select a different version.  [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605:  KafkaFlow.IntegrationTests -> KafkaFlow -> Confluent.Kafka 2.1.1 -> librdkafka.redist (>= 2.1.1)  [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605:  KafkaFlow.IntegrationTests -> librdkafka.redist (>= 1.9.2) [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605: Warning As Error: Detected package downgrade: librdkafka.redist from 2.1.1 to 1.9.2. Reference the package directly from the project to select a different version.  [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605:  KafkaFlow.IntegrationTests -> KafkaFlow.Serializer.SchemaRegistry.ConfluentJson -> Confluent.SchemaRegistry 2.1.1 -> Confluent.Kafka 2.1.1 -> librdkafka.redist (>= 2.1.1)  [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj : error NU1605:  KafkaFlow.IntegrationTests -> librdkafka.redist (>= 1.9.2) [/Users/guilherme.ferreira/Documents/GitHub/kafka-flow/src/KafkaFlow.sln]
```

### KafkaFlow version

v2.5.0

--- a/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj
+++ b/src/KafkaFlow.IntegrationTests/KafkaFlow.IntegrationTests.csproj
@@ -28,11 +28,6 @@
 		<PackageReference Include="OpenTelemetry.Exporter.InMemory" Version="1.6.0" />
 		<PackageReference Include="Polly" Version="8.0.0" />
     </ItemGroup>
-    
-    <ItemGroup Condition="$([MSBuild]::IsOsPlatform('OSX'))">
-        <PackageReference Include="librdkafka.redist" Version="1.9.2" />
-    </ItemGroup>
-    
 
     <ItemGroup>
         <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj" />

